lib/User/Backend.php

```diff
@@ -147,6 +154,11 @@ public function getCurrentUserId() {
 			return '';
 		}
 
+		if ($this->providerService->getSetting($provider->getId(), ProviderService::SETTING_CHECK_BEARER, '0') !== '1') {
+			$this->logger->error('Bearer token check is disabled for provider ' . $provider->getId());
```

Comment:
```suggestion
			$this->logger->debug('Bearer token check is disabled for provider ' . $provider->getId());
```

Otherwise this would spam heavily for disabled checking

Reply:
Amended to the existing commit.